src/test/java/games/strategy/test/EqualsPredicateTests.java

```diff
@@ -105,7 +108,8 @@ public void shouldReturnFalseWhenObjectsAreNotEqual() {
           new FakeNestedClass(new FakeClass(-42))));
     }
 
-    private static final class FakeNestedClass {
+    @Nested
+    private final class FakeNestedClass {
```

Comment:
This class does not contain tests, and thus does not need to be annotated with `@Nested`.